.github/workflows/ci.yml

```diff
@@ -18,12 +18,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Setup JDK
-        uses: actions/setup-java@v4
-        with:
-          distribution: corretto
-          java-version: 11
```

Comment:
Since [.tool-versions](https://github.com/guardian/scrooge-extras/blob/main/.tool-versions) specifies Java 21, this is a change in the Java version used for compilation from 11 to 21, which I reckon requires a bit more testing. I think we should keep the version at 11 in this PR by leaving setup-java as-is and adding setup-sbt, instead of using setup-scala. (Once we've updated this to Java 21 we can then move to setup-scala.)